### ISSUE TITLE
Question/Feature Request: Cell to be vertically or horizontally split into more small cells

### ISSUE THREAD

**Issue by FlyingDutchmanLQY** [vandeseer/easytable#167]
Hello!
Can a cell be split vertically or horizontally into more small cells?
Or multiple cells merged into big one like excel does?

something like below
![image](https://github.com/vandeseer/easytable/assets/46863356/f87c7f01-6f90-44a7-beeb-7cf1e889c77d)

**Comment by vandeseer** [comment]
Hi, 

Yes, you can merge cells. Please check the examples, there you can also find some code for it.

Theoretically you can also create tables within single cells, but I wouldn't necessarily recommend that approach. 

Best 
Stefan

**Comment by FlyingDutchmanLQY** [comment]
Sorry, bad English. Didn't fully understand the meaning of 'span'.
The example is great. But with regard to merge, can I merge cells after all the cells and rows being added, which means after the table builder finishes building. 
I have to add cells and rows in a loop and after that merge some of the cells.

**Comment by vandeseer** [comment]
No, that doesn't work. It also would render the builder useless. You would need to keep track of the cells to be merged **before** building. 

Best,
Stefan
